packages/combobox/src/Combobox.ts

```diff
@@ -378,6 +415,7 @@ export class Combobox extends Textfield {
                 ?required=${this.required}
                 ?readonly=${this.readonly}
             />
+            ${when(this.pending, () => this.renderLoader())}
```

Comment:
```suggestion
            ${when(this.pending, this.renderLoader)}
```
This is much faster

Reply:
I used arrow function because I use `this` inside `renderLoader` and I need the _outer this_ context. I can however change it to `this.renderLoader.bind(this)`